objects/server/server.go

```diff
@@ -58,7 +64,7 @@ func (b *buildServerCmd) flags() []cli.Flag {
 // * Generate a Dockerfile that gets all the extensions (using dep)
 // • ‎then generate a main.go with extensions
 // • ‎compile, throw in another container like main dockerfile
-func (b *buildServerCmd) buildServer(c *cli.Context) error {
+func (b *buildServerCmd) Build(c *cli.Context) error {
```

Comment:
Don't see this referenced  anywhere 

Suggest you put this as its own top-level command rather than under the build verb 

`fn build-server` ?